tools/kmod/sof_insert.sh

```diff
@@ -72,17 +72,19 @@ insert_module snd_soc_max98390
 # insert top-level ACPI/PCI SOF drivers. They will register SOF components and
 # load machine drivers as needed. Do not insert any other sort of audio module,
 # code dependencies will be used to load the relevant modules.
+insert_module snd_sof_acpi
+
 insert_module snd_sof_acpi_intel_byt
 insert_module snd_sof_acpi_intel_bdw
 
+insert_module snd_sof_pci # ipc_type=1 sof_pci_debug=1 ...
```

Comment:
we already pass parameters with options snd_sof_pci ipc_type=1 in a /etc/modprobe.d/something.conf

That could conflict or become inconsistent. I am not sure we want to add module parameters here, we only wanted to manage the dependencies.

To your point though it makes no sense to add them last, I think this is a left-over from earlier versions before we flipped the dependencies.

Reply:
This is only a quick hack useful locally, it's never meant to be committed and used in production. I will add a comment to clarify this.

It's useful locally because it shows up in "git diff" whereas changes in /etc/ are usually forgotten.

We already had this discussion in #734 :-)